File: Algorithms/LinearSearch/LinearSearch.go
package main

import "fmt"

func linearSearch(arr []int, target int) int {
	for i, v := range arr {
		if v == target {
			return i
		}
	}
	return -1
}

func main() {
	nums := []int{10, 20, 30, 40}
	fmt.Println(linearSearch(nums, 30))
}

File: Algorithms/LinearSearch/LinearSearch.php
<?php

function linearSearch(array $arr, int $target): int {
    foreach ($arr as $index => $value) {
        if ($value === $target) {
            return $index;
        }
    }
    return -1;
}

$numbers = [10, 20, 30, 40];

$result = linearSearch($numbers, 30);
echo $result . PHP_EOL;